### PR TITLE
Bump Ruby from 2.6.5 to 2.7.0

### LINKED ISSUE #125: Update Ruby to 2.7
Ruby 2.7 has been released.
https://www.ruby-lang.org/en/news/2019/12/25/ruby-2-7-0-released/

The current version:
https://github.com/sider/devon_rex/blob/d2fbe876a8b1b0991a9e1e03bf29b5054f6cf36e/.ruby-version#L1

### LINKED ISSUE #142: Remove rbenv
We no longer need [rbenv](https://github.com/rbenv/rbenv) to support multiple Ruby versions.

### PR DESCRIPTION
**Pull request by ybiquitous**
- News: https://www.ruby-lang.org/en/news/2019/12/25/ruby-2-7-0-released
- Official Docker image: https://hub.docker.com/_/ruby/
- Remove unneeded rbenv. Instead, use the standalone [ruby-build](https://github.com/rbenv/ruby-build).

Fix #125 
Fix #142

### CODE REVIEW

**Comment by ybiquitous** [comment]
[note] The comparison between Ruby Docker images:

- Docker team: https://github.com/docker-library/ruby
  - Star: 396
  - The latest commit: Jan 4, 2020
- Ruby team: https://github.com/ruby/ruby-docker-images
  - Star: 18
  - The latest commit: Dec 18, 2019

I decided that **docker-library/ruby** is more active.

**Comment by ybiquitous** [comment]
Your concern makes sense. 👍 

If we need to ensure the same Ruby build for all devon_rex images, we should give up using the official Ruby image.
On the other hand, using the official Ruby image can reduce the CI build time.

**Comment by ybiquitous** [comment]
I think we cannot accurately predicate what impact the difference which you point out will have.
So, it seems good to me to give up the official image if we need safety for the runtime environments.
What do you think? @yykamei

**Comment by ybiquitous** [comment]
@yykamei Wait, I will remove the part of using the official image from this PR. No problem?

**Comment by yykamei** [comment]
> Wait, I will remove the part of using the official image from this PR. No problem?

Sure, I'm waiting :smile:

File: go/Dockerfile.erb [hunk 1]
```diff
@@ -3,11 +3,12 @@ FROM golang:1.14.0-buster
 <%= ERB.new(File.read('base/Dockerfile.env.erb')).result %>
 <%= ERB.new(File.read('base/Dockerfile.prepare.erb')).result %>
 
+RUN /tmp/devon_rex_work/build-ruby
```
Comment (yykamei) [inline go/Dockerfile.erb]: This instruction could be placed in `Dockerfile.erb`?

**Comment by ybiquitous** [comment]
Thank you for the review!

**Pull request merged**